internal/pkg/cli/flag.go

```diff
@@ -18,6 +18,7 @@ const (
 	// Command specific flags.
 	dockerFileFlag        = "dockerfile"
 	imageTagFlag          = "tag"
+	awsTagsFlag           = "tags"
```

Comment:
This might be very confusing. Maybe `resource-tags`? I

Reply:
Yeah, my plan was to make a breaking change and rename `imageTagFlag = "image-tag"` but I like your suggestion more! This way no breaking change would be introduced.